app/src/main/java/com/github/se/polyfit/ui/theme/Color.kt

```diff
@@ -9,3 +9,5 @@ val Pink80 = Color(0xFFEFB8C8)
 val Purple40 = Color(0xFF6650a4)
 val PurpleGrey40 = Color(0xFF625b71)
 val Pink40 = Color(0xFF7D5260)
+val BlueButton = Color(0xFF7158E2) // I quickly to match the figma, I don't know which material it is.
+val PinkTitle = Color(0xFFCD84F1)
```

Comment:
Might want to name these better. The blue/purple and pink are both in buttons, bottom navigation bars and other parts. Maybe something like primaryPurple?